R/annotation-logticks.r

```diff
@@ -200,7 +200,10 @@ GeomLogticks <- ggproto("GeomLogticks", Geom,
     gTree(children = do.call("gList", ticks))
   },
 
-  default_aes = aes(colour = "black", size = 0.5, linetype = 1, alpha = 1)
+  default_aes = expr(aes(
+    colour = theme$geom$colour, size = 0.5, linetype = 1,
```

Comment:
I think `theme$geom$colour` is a bit long and a bit magical. I think it would be better to have a function like `foo("colour")`, but I don't know what we'd call it.